portfolio/src/main/webapp/style.css

```diff
@@ -14,7 +14,7 @@ body {
 }
 
 /* Style the content */
-#content p { 
+#content { 
```

Comment:
This change means that this formatting will apply to everything inside of `#content`, not just the paragraphs inside `#content`. I just wanted to double check that that's what you intended to do.